build/index.js

```diff
@@ -18,6 +19,18 @@ exports.onCreateProject = function (api, app, config, cb) {
     return fs.outputFileAsync(path.join(outpath, file), xmlDoc.toString(), 'utf-8');
   };
 
-  return copyCustomRules(config.outputPath)
-    .then(cb);
-}; 
+  if (config.target === 'native-android') {
+    return copyCustomRules(outputPath)
+      .then(cb);
+  } else if (config.target === 'native-ios') {
+    var buildScipt = path.join(__dirname, '../ios', 'buildScript'),
+      manifest = app.manifest.ios,
+      scriptContent = fs.readFileSync(buildScipt, 'utf-8');
+
+    outputPath = path.join(outputPath, 'xcodeproject');
+    return fs.outputFileAsync(path.join(outputPath, 'resources', 'extra'),
+      scriptContent.toString() + ' ' +manifest.crashlyticsKey +
+        ' ' + manifest.crashlyticsExtraKey, 'utf-8')
```

Comment:
what will be crashlyticsExtraKey ?

Reply:
crashlyticsExtraKey is extra key assigned to run the script with the crashlytics app key.